core/imageroot/var/lib/nethserver/cluster/actions/add-node/50update

```diff
@@ -85,13 +85,6 @@ agent.assert_exp(rdb.hset(f'node/{node_id}/vpn', mapping={
     "listen_port": str(listen_port),
 }) >= 0)
 
-#
-# 1/d. Initialize the node agent environment
-#
-agent.assert_exp(rdb.hset(f'node/{node_id}/environment', mapping={
-    "NODE_ID": node_id
-}) >= 0)
-
```

Comment:
Why this check has been removed? Redis should already contain the information.

Reply:
My bad, we should not write anymore on redis env, this is why it has been removed: it's ok.